modules/pensions/lib/pensions/pdf_fill/sections/section_03.rb

```diff
@@ -7,7 +7,122 @@ module PdfFill
     # Section III: Veteran Service Information
```

Comment:
Just as a note here: There isn't any net new code, this is all just moving code from one file to other files